### PR TITLE
Add link to PolygonScan from wallet address

### PR DESCRIPTION
**Pull request by slinlee**
- Detect Polygon ChainID in MetaMask
- Use polygon FWEB3 tokens
- third dot
- polygon contract
- notion
- UX improvements
- cp
- 3rd dot working via polygonscan
- cp
- cp
- cp
- fourth dot
- skeleton of sharing code
- cp
- cp
- cp
- trim share
- cp
- cp
- view wallets
- fix for issue #32
- remove console.log
- check that wallet address is sender to complete dot 4
- fixes issue 36 by exiting function before invoking polygonscan api if wallet address is undefined
- cp
- fweb3 diamond nft smart contract
- cp
- cp
- cp
- cp
- cp
- cp
- cp
- cp
- cp
- Add example .env.local file
- Implements #12 Tracks burned tokens
- Filter to the senders address too
- Apply Prettier
- Remove .env.local.example
- Add .env.local.example
- Make first dot pulse if not connected to metamask
- Show the first dot if `wallet` query param is provided
- Don't show message about polygon if `wallet` query param is provided
- WIP - get token balance from Polygon
- Refactor token balance in UI
- Remove unused file
- Fix paren
- cp
- mintable
- cp
- cp
- cp
- cp
- cp
- nft fixes
- cp
- cp
- cp
- two column layout
- cp
- cp
- cp
- cp
- round number
- cruft deletion
- cp
- cp
- nft fixes and new contract
- cp
- cp
- cp
- remove refs to feb
- ninth dot
- poll
- cp
- cp
- cp
- cp
- cp
- cp
- cp
- cp
- cp
- cp
- cp
- cp
- cp
- cp
- game smart contract
- cp
- cp
- cp
- cp
- Add Cypress framework for tests
- Apply prettier
- Add more basic tests for empty and completed wallet
- update token count
- Delete example fixtures
- Remove more example files
- Add tests instructions to README
- Fix first dot when using query address
- Fix the token balance when using query wallet
- Enable the test for a completed wallet
- Don't call polygon endpoint multiple times
- cp
- cp
- cp
- cp
- cp
- cp
- mapping
- mapping
- cp
- cp
- cp
- cp
- cp
- trophy attempt
- cp
- Rename polygon.js -> polygon.ts
- Apply Prettier formatting
- Fix `==` comparisons
- Add initial tests for our polygon endpoint
- Format with prettier
- cp - adding check for proper body response
- remove rejection
- cp
- Update tests
- Apply code conventions and formatting
- Remove debugging log
- cp
- add prettier to project for code formatting consistency
- Add link to polygonscan for wallet addresses

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ export default function Home() {
         </p>
 
         {query.wallet !== undefined && query.wallet !== account && query.wallet.length > 0 && (
-          <p style={{color: "#fff"}}>{query.wallet}</p>
+          <a href={"https://polygonscan.com/address/" + query.wallet} target="_blank" rel="noreferrer"><p style={{color: "#fff"}}>{query.wallet}</p></a>
```
Comment (slavingia) [inline pages/index.tsx:131]:
```suggestion
          <a href={"https://polygonscan.com/address/" + query.wallet} target="_blank" rel="noreferrer">
            <p style={{color: "#fff"}}>
              {query.wallet}
            </p>
          </a>
```
